consensus/src/ledger.rs

```diff
@@ -209,14 +209,31 @@ impl OperationLedgerInterface for Operation {
                     &LedgerChange {
                         balance_delta: roll_price
                             .checked_mul_u64(*roll_count)
-                            .ok_or(ConsensusError::RollOverflowError)?,
+                            .ok_or(ConsensusError::AmountOverflowError)?,
                         balance_increment: false,
                     },
                 )?;
             }
             // roll sale is handled separately with a delay
             models::OperationType::RollSell { .. } => {}
-            models::OperationType::ExecuteSC { .. } => {}
+            models::OperationType::ExecuteSC {
+                max_gas,
+                gas_price,
+                coins,
+                ..
+            } => {
+                res.apply(
```

Comment:
Why we don't credit the fee recipients here ?

Reply:
they will be credited in the SCE